### ISSUE TITLE
High loss when training coco dataset using multi gpu

### ISSUE THREAD

**Issue by yacad** [AlexeyAB/darknet#7176]
Hi @AlexeyAB

I want to train a coco dataset using multiple-GPUs(RTX3090 * 3) and compare it with your yolov4.weights file.
I made the yolov4-multi.cfg file by copying the yolov4.cfg file and modified it as follows:

[net]
batch=64
subdivisions=8
width=416
height=416
channels=3
momentum=0.949
decay=0.0005
angle=0
saturation = 1.5
exposure = 1.5
hue=.1

learning_rate=0.00087
burn_in=3000
max_batches = 500500
policy=steps
steps=400000,450000
scales=.1,.1

Only the width, height, learning_rate and burn_in are changed, and the rest are the same as the yolov4.cfg file.
I set the values of learning_rate and burn_in by looking at "How to train with multi-GPU".
[[ If you get a Nan, then for some datasets better to decrease learning rate, for 4 GPUs set learning_rate = 0,00065 (i.e. learning_rate = 0.00261 / GPUs). In this case also increase 4x times burn_in = in your cfg-file. I.e. use burn_in = 4000 instead of 1000. ]]

Train it first on 1 GPU for like 1000 iterations: 
$ ./darknet detector train cfg/coco.data cfg/yolov4-multi.cfg yolov4.conv.137
I did it over 1000 iterations but couldn't get the yolov4-multi_1000.weights file. Perhaps it is due to the large value of max_batches.

So I copied the yolov4-multi.cfg file to create a yolov4-multi-1000.cfg file and modified it as follows:

learning_rate=0.00261
burn_in=1000
max_batches = 1000
policy=steps
steps=800,900

Train it first on 1 GPU for like 1000 iterations: 
$ ./darknet detector train cfg/coco.data **cfg/yolov4-multi-1000.cfg** yolov4.conv.137
Then I renamed the final weights file to yolov4-multi_1000.weights. And I used the yolov4-multi_1000.weights file to train on multiple-GPUs.
$ ./darknet detector train cfg/coco.data **cfg/yolov4-multi.cfg yolov4-multi_1000.weights** -gpus 0,1,2 -map
Then, training is in progress with a very high avg-loss as shown in the chart below.
![image](https://user-images.githubusercontent.com/53897615/103128521-0d4e1280-46d9-11eb-85bc-7ad364d7aa41.png)

I have a question about it.
1. Should I create a separate cfg(max_batches = 1000) file like me first learning on one GPU for 1000 iterations? If not I've iterated over 1000 times but I can't get the yolov4-multi_1000.weights file(Perhaps it is due to the large value of max_batches). How can I create a yolov4-multi_1000.weights file?
2. Looking at the chart above, is the training progressing normally?
3. You said that max_batches is classes * 2000, but if you look at yolov4.cfg you set a much larger 500,500. Is there a reason? If I train using the modified yolov4-multi.cfg, can I get the same result as you?
4. Can you share the charts and cfg files from yolov4.weights?
